elixir/apps/api/mix.exs

```diff
@@ -43,7 +43,6 @@ defmodule API.MixProject do
       {:domain, in_umbrella: true},
 
       # Phoenix deps
-      {:phoenix, "~> 1.7.0"},
```

Comment:
We don't need child app overrides since they are the same version.

Reply:
Turns out these were still needed in the built applications because of how they're packaged.